content/en/agent/kubernetes/_index.md

```diff
@@ -255,6 +254,100 @@ For further details on setting up Operator, including information about using to
       fsGroup: <DOCKER_GROUP_ID>
 ```
 
+## Kubernetes resources for Live Containers
+
+The Agent and [Cluster Agent][3] can be configured to retrieve Kubernetes resources for [Live Containers][4]. This feature allows you to monitor the state of pods or deployments in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires the following set up prior to configuration:
+
+- Agent version >= 7.21.1
+- Process-agent is running (it doesn't need to run the process collection), and configured with the following options:
+
+  {{< code-block lang="yaml">}}
+  - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+    value: "true"
+  - name: DD_ORCHESTRATOR_CLUSTER_ID
+    valueFrom:
+    configMapKeyRef:
+      name: datadog-cluster-id
+      key: id
+  {{< /code-block >}}
+
+  It must also be configured to communicate with the Cluster Agent, see the [Cluster Agent Setup documentation][5] for more information about this.
+
+- Cluster Agent version >= 1.9.0
+- Cluster Agent is running, and the Agent must be able to communicate with it
+
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
+- Set `datadog.orchestratorExplorer.enabled` to true in [values.yaml][2]
+- Deploy a new release
+
+
+[1]: https://github.com/DataDog/helm-charts
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "DaemonSet" %}}
+
+Update the Agent DaemonSet, the Cluster Agent Deployment, and the Cluster Agent ClusterRole with the following settings:
+
+## Cluster Agent Settings
```

Comment:
It may look like a nitpick, but can we move the cluster agent settings above the agent settings (after we bring the agent settings in this DaemonSet tab) please? I'm worried that some users do the agent change and try to deploy without reading the cluster agent changes, and this will block their process-agent from starting, which i'd rather avoid.